### PR TITLE
UX Bucket summary

### PR DESCRIPTION
**Pull request by prakashsvmx**
- UX Bucket summary
- Refactor Bucket Tags to be  a separate component 
- Reduce reliance on the classes being passed down.

UX Spec:
https://xd.adobe.com/view/2b227c2c-2e5f-46ee-8d20-d90cc76ee8b4-7e71/specs/

<summary> How does it look ? </summary>

<details>

#### All features
image

#### Default features

image

</details>

Note: Sidebar menu UX will be sent in the upcoming PR.

### CODE REVIEW

**Review by dvaldivia — DISMISSED** [review]

File: portal-ui/src/icons/DisabledIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+// This file is part of MinIO Console Server
+// Copyright (c) 2021 MinIO, Inc.
```
Comment (harshavardhana) [inline portal-ui/src/icons/DisabledIcon.tsx:2]: New files should have 2022 now @prakashsvmx

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: portal-ui/src/icons/DisabledIcon.tsx (hunk 1 above)
Comment (dvaldivia) [reply portal-ui/src/icons/DisabledIcon.tsx:2]: this PR is from last year @harshavardhana xD

Comment (harshavardhana) [reply portal-ui/src/icons/DisabledIcon.tsx:2]: yeah but its not merged yet @dvaldivia lets update the 2021 -> 2022

Comment (prakashsvmx) [reply portal-ui/src/icons/DisabledIcon.tsx]: Thank you @harshavardhana @dvaldivia . I have updated the year for the new files.

**Review by dvaldivia — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**